src/test/kotlin/com/kotlin/boardproject/service/PostServiceImplTest.kt

```diff
@@ -1,28 +1,30 @@
 package com.kotlin.boardproject.service
 
-import com.kotlin.boardproject.auth.ProviderType
-import com.kotlin.boardproject.common.enums.ErrorCode
-import com.kotlin.boardproject.common.enums.NormalType
-import com.kotlin.boardproject.common.enums.PostStatus
-import com.kotlin.boardproject.common.enums.Role
-import com.kotlin.boardproject.common.exception.ConditionConflictException
-import com.kotlin.boardproject.common.exception.EntityNotFoundException
-import com.kotlin.boardproject.common.exception.UnAuthorizedException
-import com.kotlin.boardproject.dto.post.normalpost.CreateNormalPostRequestDto
-import com.kotlin.boardproject.dto.post.normalpost.CreateNormalPostResponseDto
-import com.kotlin.boardproject.dto.post.normalpost.EditNormalPostRequestDto
-import com.kotlin.boardproject.dto.post.normalpost.EditNormalPostResponseDto
-import com.kotlin.boardproject.model.LikePost
-import com.kotlin.boardproject.model.NormalPost
-import com.kotlin.boardproject.model.ScrapPost
-import com.kotlin.boardproject.model.User
-import com.kotlin.boardproject.repository.BasePostRepository
-import com.kotlin.boardproject.repository.BlackPostRepository
-import com.kotlin.boardproject.repository.CommentRepository
-import com.kotlin.boardproject.repository.LikePostRepository
-import com.kotlin.boardproject.repository.NormalPostRepository
-import com.kotlin.boardproject.repository.ScrapPostRepository
-import com.kotlin.boardproject.repository.UserRepository
+import com.kotlin.boardproject.global.enums.ProviderType
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end